MvcChatBot.Agent/Services/TwitchClientService.cs

```diff
@@ -103,7 +104,10 @@ private void Client_MessageReceived(object sender, OnMessageReceivedArgs e)
                 }
             }
 
-
+            if(e.ChatMessage.Message.StartsWith("!stats"))
```

Comment:
This should live in the on_command received switch - see line 132ish - then call a private method.